packages/experimental/Tabs/src/TabsItemTokens.ts

```diff
@@ -0,0 +1,49 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { TabsItemTokens } from '.';
+
+export const tabsItemStates: (keyof TabsItemTokens)[] = ['hovered', 'selected', 'focused', 'disabled', 'pressed'];
+
+export const defaultTabsItemTokens: TokenSettings<TabsItemTokens, Theme> = (t: Theme) =>
+  ({
+    backgroundColor: 'transparent',
+    color: t.colors.neutralForeground3Brand,
```

Comment:
I believe some of these were updated, so you might want to double check some of the color entries here.

Reply:
updated now!